qcengine/programs/nwchem/germinate.py

```diff
@@ -61,6 +61,9 @@ def muster_modelchem(method: str, derint: int, use_tce: bool) -> Tuple[str, Dict
             mdccmd = f"task tce {runtyp}\n\n"
             opts["tce__ccsd(t)"] = True
         else:
-            mdccmd = f"task ccsd(t) {runtyp}\n\n"
+            mdccmd = f'task ccsd(t) {runtyp}\n\n'
```

Comment:
just a note that they'll probably roll out `black` which is a different formatter from `yapf`, and it changes all your `'` to `"`.